pkg/gcptarget/library.go

```diff
@@ -19,7 +19,6 @@ import "text/template"
 /*
 This comment puts the start of rego on line 10 so it's easier to do math when
 it calls out the line number.
-
```

Comment:
If I understand correctly - the library.go files are no longer used. Would it be safe to fully delete them?

Reply:
Removed, and also removed the Library() function in gcptarget.go and tftarget.go